openshift/system/config/prometheus.yml

```diff
@@ -1,3 +0,0 @@
-production:
```

Comment:
This files is not used. It was initially added here: https://github.com/3scale/porta/commit/809e5dedfda87c1ae01ea37d5e91c58d8bff2b0a#diff-c1fd91cb1911a0512578b99f657554526f3e1421decdb9e908712beab57e10f9 , but then its loading was removed in https://github.com/3scale/porta/commit/79d15baae271c7e568ed351d9c41f5f6263b0324